configurations/csbc-pson/routesConfig.ts

```diff
@@ -0,0 +1,133 @@
+import { GenericRoute } from '../types/portal-config'
+import { publications, data, datasets, grants, studies } from './homeExploreConfiguration'
+import { SynapseConstants } from 'synapse-react-client'
+import { studiesSql } from './homeExploreConfiguration/studies'
+import { publicationSql } from './homeExploreConfiguration/publications'
+import { datasetsSql } from './homeExploreConfiguration/datasets'
+
+const homeLimit = 3
+
+const routes: GenericRoute [] = [
+  {
+    name: 'Home',
+    to: '/',
+    isNested: false,
+    synapseObject: [
+      {
+        name: 'StackedBarChartPreview',
+        title: 'EXPLORE PORTALS',
+        props: {
+          queryWrapperConfigs: [
+            {
+              ...grants.homePageSynapseObject.props
```

Comment:
could simplify (instead of expanding props into a new dictionary that contains the same content)